backend/app/views/spree/admin/payments/credit.html.erb

```diff
@@ -1,18 +0,0 @@
-<%= render partial: 'spree/admin/shared/order_tabs', locals: { current: 'Creditcards' } %>
```

Comment:
Right! I gave details about this removal in the commit (0192bc00cc41099aa650c1b6818514686e3e24cd) but I forgot to stress in the PR description that I needed you to confirm we could to this. 

While moving the breadcrumb logic to controllers, I found that this template didn't seem to be used. Basically, I couldn't find any route that would use it or any other code referencing it. Also, no test broke after removing this template. But, of course, you know the codebase better than me, and you need to confirm it's ok to remove it.

Reply:
Yes, I did read that commit message (thanks for being so detailed there). My face was because I didn't expect to have an unused file in the application!